apps/checker/test/scala/matchers/LanguageToolMatcherTest.scala

```diff
@@ -94,19 +96,19 @@ class LanguageToolMatcherTest extends AsyncFlatSpec with Matchers {
       </rule>
   </rulegroup>
   """
-  val exampleRulegroup =  LTRuleXML(
+  val exampleRulegroup =  typerighter.model.LTRuleXML(
     "EXAMPLE_RULEGROUP",
     exampleRulegroupXml,
     exampleCategory,
     "An example rulegroup with custom XML"
   )
-  val exampleBadRule1 = LTRuleXML(
+  val exampleBadRule1 = typerighter.model.LTRuleXML(
```

Comment:
Is this required?
```suggestion
  val exampleBadRule1 = LTRuleXML(
```

Reply:
Thanks for spotting! [4e54ff6](https://github.com/guardian/typerighter/pull/239/commits/4e54ff6739e8ccff015950feceb0b17ffda306f9)